docs/development/setup_development.rst

```diff
@@ -20,21 +20,36 @@ Quick Start
 The Docker based environment is suitable for developing the web application
 and updating the documentation.
 
+Ensure Git is installed on your system. Use package management tools to update your local package index. Once the update is complete, you can download and install Git:
 
 Ubuntu or Debian GNU/Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
+Run the following commands to update the packages and to install from the default Git repositories:
```

Comment:
I would shorten to:
"Run the following commands to update the package index and to install Git"

Technically they are not Git repositories but distribution repositories, but we don't have to spell that out either way.